### PR TITLE
feature/FOUR-19330

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
Made the change to call the new API `/requests-by-case?case_number=` 
At the same time, work was done on adapting some columns to correctly receive the data from the API and changing the table to perform the sort

FOUR-18139: Enable the order desc/asc "Requests" tab

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19330

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-3742593f62.engk8s.processmaker.net

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 1]
```diff
@@ -66,6 +70,7 @@ const requestNumberColumn = () => ({
   field: "id",
   header: "Request #",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:71]: Please update to `Request ID`
image

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 2]
```diff
@@ -78,10 +83,11 @@ const requestNumberColumn = () => ({
 });
 
 const requestNameColumn = () => ({
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:85]: This column does not exist
image

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 3]
```diff
@@ -93,23 +99,41 @@ const requestNameColumn = () => ({
...
+    params: {
+      click: (option, row, column, columns) => {
+        window.document.location = `/tasks/${option.id}/edit`;
+      },
+      formatterOptions:(option, row, column, columns)=>{
+        return option.element_name;
+      }
+    },
+  }),
 });
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:121]: The column participants is missing
image

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/jscomposition/cases/casesDetail/api/index.js [hunk 4]
```diff
@@ -1,25 +1,14 @@
...
-      process_name: `Process ${i}`,
-      assigned: `Avatar ${i}`,
-      current_task: `Task ${i}`,
-      status: `badge ${i}`,
-      started: `21/21/${i}`,
-      due_date: `21/21/${i}`,
-    };
-
-    objectsList.push(obj);
-  }
```
Comment (fagubla) [inline resources/jscomposition/cases/casesDetail/api/index.js:20]: we still using this method

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-3742593f62.engk8s.processmaker.net

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 5]
```diff
@@ -64,9 +68,10 @@ const dueDateColumn = () => ({
 // Columns for Requests
 const requestNumberColumn = () => ({
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:69]: update to requestIdColumn

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 6]
```diff
@@ -77,11 +82,12 @@ const requestNumberColumn = () => ({
   }),
 });
 
-const requestNameColumn = () => ({
-  field: "case_title",
-  header: "Request Name",
+const processNameColumnRequest = () => ({
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:85]: update to `processNameColumn`

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 7]
```diff
@@ -93,23 +99,41 @@ const requestNameColumn = () => ({
 });
 
 const currentTaskColumn = () => ({
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:101]: Update to `TaskColumn`

File: resources/jscomposition/cases/casesDetail/config/columns.js [hunk 8]
```diff
@@ -93,23 +99,41 @@ const requestNameColumn = () => ({
...
+  width: 140,
+  cellRenderer: () => ({
+    component: StatusCell,
+  }),
 });
 
 const startedColumn = () => ({
-  field: "started_date",
+  field: "initiated_at",
   header: "started",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/config/columns.js:135]: Please update to `Started`

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Review by pmPaulis — APPROVED** [review]

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**